docs/dev-guide/changes.md

```diff
@@ -0,0 +1,231 @@
+or code to utilize, as well as breaking changes that may cause existing model
+configurations or code to fail to run correctly.
+
+## v1.3
+
+### Logging
+
+The reading of YAML configuration files has been modified to use the "safe" reader,
+which prohibits the use of arbitrary Python code in configuration files. This is a
+security enhancement, but it requires some changes to the way logging is configured.
+
+In previous versions, the logging configuration file could contain Python code to
+place log files in various subdirectories of the output directory, which might
+vary for different subprocesses of the model, like this:
+
+```yaml
+logging:
+  handlers:
+    logfile:
+      class: logging.FileHandler
+      filename: !!python/object/apply:activitysim.core.config.log_file_path ['activitysim.log']
+      mode: w
+      formatter: fileFormatter
+      level: NOTSET
+```
+
+In the new version, the use of `!!python/object/apply` is prohibited. Instead of using
+this directive, the `log_file_path` function can be invoked in the configuration file
+by using the `get_log_file_path` key, like this:
+
+```yaml
+logging:
+  handlers:
+    logfile:
+      class: logging.FileHandler
+      filename:
+        get_log_file_path: activitysim.log
+      mode: w
+      formatter: fileFormatter
+      level: NOTSET
+```
+
+Similarly, previous use of the `if_sub_task` directive in the logging level
+configuration like this:
+
+```yaml
+logging:
+  handlers:
+    console:
+      class: logging.StreamHandler
+      stream: ext://sys.stdout
+      level: !!python/object/apply:activitysim.core.mp_tasks.if_sub_task [WARNING, NOTSET]
+      formatter: elapsedFormatter
+```
+
+can be replaced with the `if_sub_task` and `if_not_sub_task` keys, like this:
+
+```yaml
+logging:
+  handlers:
+    console:
+      class: logging.StreamHandler
+      stream: ext://sys.stdout
+      level:
+        if_sub_task: WARNING
+        if_not_sub_task: NOTSET
+      formatter: elapsedFormatter
+```
+
+For more details, see [logging](Logging).
+
+### Chunking
+
+Version 1.3 introduces a new "[explicit](Explicit-Chunking)" chunking mechanism.
+
+Explicit chunking is simpler to use and understand than dynamic chunking, and in
+practice has been found to be more robust and reliable. It requires no "training"
+and is activated in the top level model configuration file (typically `settings.yaml`):
+
+```yaml
+chunk_training_mode: explicit
+```
+
+Then, for model components that may stress the memory limits of the machine,
+the user can specify the number of choosers in each chunk explicitly, either as an integer
+number of choosers per chunk, or as a fraction of the overall number of choosers.
+This is done by setting the `explicit_chunk` configuration setting in the model
+component's settings.  If the `explicit_chunk` value is 0 or missing, then no
+chunking is applied for that component.  The `explicit_chunk` values are ignored
```

Comment:
Suggest we say here that if  explicit_chunk > 1 then it corresponds to the number of rows. If fraction, then it corresponds to the fraction of the total number of choosers.